scriptworker/test/test_artifacts.py

```diff
@@ -253,7 +253,8 @@ def test_get_upstream_artifacts_full_paths_per_task_id(context):
     }
 
 
-def test_fail_get_upstream_artifacts_full_paths_per_task_id(context):
+@pytest.mark.asyncio
+async def test_fail_get_upstream_artifacts_full_paths_per_task_id(context):
```

Comment:
I'm a little bit confused on why this function must be declared async, but not `test_get_and_check_single_upstream_artifact_full_path()`. They are both testing the same function and they both rely on the context. In the function under test, the context doesn't require to be async.

What's the reason for flipping this particular function?

Reply:
I wasn't sure what exactly was breaking - The error was that the event loop wasn't available, and this was one of the tests that showed the error. This may be because the loop wasn't available in this test, or the warning may be output from another test, but appear next to this test since the tests are run concurrently. I agree that it doesn't make sense that this test would require asyncio but other similar tests don't.

Reply:
Going to try removing these `@pytest.mark.asyncio`'s and see if that changes anything.